src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/ActionShuffler.kt

```diff
@@ -0,0 +1,85 @@
+package com.atlassian.performance.tools.jirasoftwareactions
+
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.action.BrowseProjectsAction
+import com.atlassian.performance.tools.jiraactions.api.action.CreateIssueAction
+import com.atlassian.performance.tools.jiraactions.api.action.SearchJqlAction
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.JqlMemory
+import com.atlassian.performance.tools.jiraactions.api.page.IssuePage
+import com.atlassian.performance.tools.jiraactions.api.scenario.addMultiple
+import com.atlassian.performance.tools.jirasoftwareactions.api.actions.BrowseBoardsAction
+import kotlin.reflect.KClass
+
+class ActionShuffler {
```

Comment:
It's not part of the API, so it could be `internal`, not to expose public non-API classes (it can be used by accident and expected to be compatible).